docs/src/components/Guidelines/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import styled, { css } from "styled-components";
-import { CheckCircle, CloseCircle } from "@kiwicom/orbit-components/icons";
+import { CheckCircle, CloseCircle, Check, Close } from "@kiwicom/orbit-components/icons";
```

Comment:
Check and Close are not being used, so why import them? Is this intended?

Reply:
They are used in GuidelineItemIcons (new component I added)

Reply:
Yes, but that is a different file, so they should only be imported there (and they are). So I will remove it from here while addressing the pipeline issues

Reply:
Ok, I was probably trying to use them here as well as it might be needed in some cases but ended up not figuring it out.